bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -63,6 +64,14 @@ private void processEvent(String content) {
             return;
         }
         String action = content.substring(startIndex, endIndex);
+        startIndex = content.indexOf(";data=", startIndex);
+        if (startIndex > 0) {
+            endIndex = content.lastIndexOf("}");
+            if (endIndex > 0) {
+                String data = content.substring(startIndex + 6, endIndex + 1);
```

Comment:
That would mean you include the ending }.
I assume this is not what you assume.

Reply:
Yes that is what is wanted. Do you suggest doing it another way? As I do not use the transforms very often so I would not know what makes the data the easiest to manipulate. The issue that is linked to this PR has examples of what the data looks like.